src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -144,17 +147,32 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 async def rclone_check_progress(job, proc):
+    dropbox__restricted_content = False
     while True:
         read = (await proc.stdout.readline()).decode()
-        job.logs_fd.write(read.encode("utf-8", "ignore"))
         if read == "":
             break
+        if "failed to open source object: path/restricted_content/" in read:
+            job.internal_data["dropbox__restricted_content"] = True
+            dropbox__restricted_content = True
+        job.logs_fd.write(read.encode("utf-8", "ignore"))
         reg = RE_TRANSF.search(read)
         if reg:
             transferred = reg.group(1).strip()
             if not transferred.isdigit():
                 job.set_progress(None, transferred)
 
+    if dropbox__restricted_content:
+        message = "\n" + (
+            "DropBox sync failed due to restricted content being present in one of the folders. This may include\n"
+            "copyrighted content or the DropBox manual PDF that appears in the home directory after signing up.\n"
+            "All other files were synchronized, but no deletions were performed as synchronization is considered\n"
+            "unsuccessful. Please inspect logs to determine which files are considered restricted and exclude them\n"
+            "from your synchronization. Refer to https://www.ixsystems.com/documentation/freenas/11.2/tasks.html\n"
```

Comment:
If the docs don't exist yet, it is probably best this is excluded until they are present.

```suggestion
            "from your synchronization."
```